### DISCUSSION TITLE
What is "Replaygain clipping"?

### DISCUSSION THREAD

**Discussion by FibreTTP** [jeffvli/feishin#532] [Q&A] [answered]
In the settings, there is a toggle named "Replaygain clipping". I would assume it enables something to *prevent* clipping in accordance with the "[standard](https://web.archive.org/web/20231121034954/https://replaygain.hydrogenaud.io/player_clipping.html)," though as it is worded, it seems like disabling it *enables* clipping prevention (because you're turning off the clipping...).

It would be nice if that setting could be clarified in the application.

**Comment by kgarner7** [comment, marked as answer]
This is what we call a description not populated correctly. See https://github.com/jeffvli/feishin/commit/9f86a8179f8185e02ec98836ba02a83dfee5a2b1. It's resolved now (see https://feishin.vercel.app for what it should look like)

> Prevent clipping caused by replaygain by automatically lowering the gain
